core/backend/src/annotations/FrameGeometry.ts

```diff
@@ -0,0 +1,324 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module ElementGeometry
+ */
+
+
+import { Id64 } from "@itwin/core-bentley";
+import { BackgroundFill, ColorDef, ElementGeometry, FillDisplay, GeometryParams, TextAnnotationFrameShape, TextFrameStyleProps } from "@itwin/core-common";
+import { Angle, AngleSweep, Arc3d, LineString3d, Loop, Path, Point3d, Range2d, Transform, Vector2d } from "@itwin/core-geometry";
+
+/**
+ * This namespace contains methods for computing the geometry a frame that encloses the range of a text annotation.
+ * @beta
+ */
+export namespace FrameGeometry {
+
+  /**
+   * Based on the frame style, this method will construct and append [[GeometryParams]] (for line style) and a [[Loop]] (for the frame shape) to the builder.
+   * @param builder that will be appended to in place
+   * @param frame
+   * @param range to enclose with the frame
+   * @param transform that transform the range to world coordinates
+   * @returns `true` if any geometry was appended to the builder
+  * @beta
+   */
+  export const appendFrameToBuilder = (builder: ElementGeometry.Builder, frame: TextFrameStyleProps, range: Range2d, transform: Transform, geomParams?: GeometryParams): boolean => {
```

Comment:
@ben-polinsky will documentation generator produce correct docs for functions declared like this instead of e.g. `export function(whatever)`?
@claudiaareneee was this intended to prevent weirdos from doing `FrameGeometry.appendFrameToBuilder = () => throw new Error("mwuhaha");`?
(IMO the namespace is just API clutter and you could remove it).

Reply:
Since it's broken, I change it to export them as functions.

Reply:
Done